ArcGISRuntimeSDKQt_CppSamples/EditData/ContingentValues/ContingentValues.qml

```diff
@@ -0,0 +1,210 @@
+// [WriteFile Name=ContingentValues, Category=EditData]
+// [Legal]
+// Copyright 2021 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.12
+import QtQuick.Controls 2.2
+import QtQuick.Layouts 1.3
+import Esri.Samples 1.0
+
+Item {
+    id: sampleWindow
+
+    // add a mapView component
+    MapView {
+        id: view
+        anchors.fill: sampleWindow
+
+        MouseArea {
+            anchors.fill: view
+            visible: attributePrompt.visible
+            onClicked: mouse.accepted = attributePrompt.visible
+            onWheel: wheel.accepted = attributePrompt.visible
+        }
+
+        Component.onCompleted: {
+            // Set and keep the focus on SceneView to enable keyboard navigation
+            forceActiveFocus();
+        }
+    }
+
+    // Declare the C++ instance which creates the scene etc. and supply the view
+    ContingentValuesSample {
+        id: contingentValuesSample
+        mapView: view
+    }
+
+    Control {
+        id: attributePrompt
+
+        // Expand the attributes pane depending on where the user clicks or taps
+        x: (contingentValuesSample.featureAttributesPaneXY[0] + width > sampleWindow.height ? contingentValuesSample.featureAttributesPaneXY[0] - width : contingentValuesSample.featureAttributesPaneXY[0]) ?? 0;
+        y: (contingentValuesSample.featureAttributesPaneXY[1] + height > sampleWindow.height ? contingentValuesSample.featureAttributesPaneXY[1] - height : contingentValuesSample.featureAttributesPaneXY[1]) ?? 0;
+
+        background: Rectangle {
+            color: "#fdfdfd"
+        }
+
+        visible: contingentValuesSample.featureAttributesPaneVisibe
+
+        contentItem: Column {
+            id: inputColumn
+            spacing: 5
+            padding: 10
+
+            Text {
+                text: "Status"
```

Comment:
In the sample "Status" appears to be a mandatory field that needs a value before the new point can be saved. Why is there an empty field under Status dropdown?